xai_components/utils.py

```diff
@@ -41,7 +42,10 @@ def set_defaults(component, tvb_datatype):
     for attr in tvb_datatype.declarative_attrs:
         attr_value = getattr(tvb_datatype, attr)
         attr_default = attr_value.default
-        if isclass(attr_default) and issubclass(attr_value.field_type, HasTraits):
+        print(type(attr_default))
```

Comment:
Maybe have a log here instead of the print ?

Reply:
will remove it, I don't think we actually need it